tinygrad/runtime/ops_cuda.py

```diff
@@ -49,7 +49,7 @@ class device:
   import pycuda.driver as cuda # type: ignore
   class RawCUDABuffer(RawBufferCopyInOut): # type: ignore
     def __init__(self, size, dtype): super().__init__(size, dtype, cuda.mem_alloc(size * dtype.itemsize)) # type: ignore
-    def _copyin(self, x:np.ndarray, stream:Optional[cuda.Stream]=None): cuda.memcpy_htod_async(self._buf, x, stream) # type: ignore
+    def _copyin(self, x:np.ndarray, stream:Optional[cuda.Stream]=None): cuda.memcpy_htod_async(self._buf, np.ascontiguousarray(x), stream) # type: ignore
```

Comment:
Should probably do `x.ravel()` if we don't always require a copy.

Reply:
@sekstini good idea, changed to ravel